src/helpers/format.ts

```diff
@@ -429,6 +429,8 @@ function formatJSDate(jsDate: DateTime, format: Format): FormattedValue {
           return MONTHS[jsDate.getMonth()];
         case "mmmmm":
           return MONTHS[jsDate.getMonth()].slice(0, 1);
+        case "q":
+          return _t("Q%(quarter)s", { quarter: jsDate.getQuarter() });
```

Comment:
Regarding your comment in the commit message, how about having a format "q" => 3 and another format "qq" => Q3 (or uppercase "Q") to have flexibility ?

Reply:
If we add it, we should add `qqqq` for `Quarter 3`.